app/apollo/resolvers/channel.js

```diff
@@ -952,14 +956,16 @@ const channelResolvers = {
           return {
             name: subscription.name,
             uuid: subscription.uuid,
+            groups: subscription.groups
           };
         });
 
         // Allow graphQL plugins to retrieve more information. removeChannelVersion can delete versions, and subscriptions. Include details of each deleted resource in pluginContext.
-        context.pluginContext = {channel: {name: channel.name, uuid: channel.uuid}, version: {name: deployableVersionObj.name, uuid: deployableVersionObj.uuid}, subscriptions: subscriptionObjs};
+        context.pluginContext = {channel: {name: channel.name, uuid: channel.uuid, tags: channel.tags}, deleted_version: {name: deployableVersionObj.name, uuid: deployableVersionObj.uuid, description: deployableVersionObj.description}, deleted_subscriptions: subscriptionObjs};
```

Comment:
Use `version` rather than `deleted_version` for consistency -- the plugin handler can differentiate by the action taken without needing specially named attributes.
```suggestion
        context.pluginContext = {channel: {name: channel.name, uuid: channel.uuid, tags: channel.tags}, version: {name: deployableVersionObj.name, uuid: deployableVersionObj.uuid, description: deployableVersionObj.description}, subscriptions: subscriptionObjs};
```